Biomass_core.R

```diff
@@ -873,8 +896,8 @@ Init <- function(sim, verbose = getOption("LandR.verbose", TRUE)) {
     maxBiomass <- maxValue(sim$biomassMap)
     if (maxBiomass < 1e3) {
       if (verbose > 0) {
-        message(crayon::green("  Because biomassMap values are all below 1000, assuming that these should be\n",
-                              "    converted to tonnes/ha by multiplying by 100"))
+        message(crayon::green("  Because biomassMap values are all below 1000, assuming that these are on ton/ha.\n",
```

Comment:
can we change ton/ha to Mg/ha (or at least tonne/ha)?

Reply:
actually we use tonnes/ha below this, so if we use Mg we should update both instances.

Reply:
tonnes/ha now